pkg/api/types.go

```diff
@@ -188,6 +188,8 @@ const (
 	NamespaceNone string = ""
 	// NamespaceSystem is the system namespace where we place system components.
 	NamespaceSystem string = "kube-system"
+	// NamespacePublic is the namespace where we place public info (ConfigMaps)
+	NamespacePublic string = "kube-public"
```

Comment:
Have you added the code that automatically creates this namespace like I did in: https://github.com/kubernetes/kubernetes/pull/25196?

I assume that since we have this namespace as a core feature, it should be created on all clusters.
We might want to have it behind a feature gate maybe, but at least it would be annoying to have to create the namespace in kubeadm or manually.